packages/zendesk-adapter/src/config.ts

```diff
@@ -1643,9 +1643,9 @@ export const DEFAULT_TYPES: ZendeskApiConfig['types'] = {
   },
   articles: {
     request: {
-      url: '/api/v2/help_center/{locale}/articles',
+      url: '/api/v2/help_center/categories/{category_id}/articles?include=translations&sort_by=updated_at',
```

Comment:
nit: let's add a comment explaining that we are doing this for better parallelization of requests on large accounts?
also - do we need the `sort_by` field?

Reply:
actually - can we do this with query params instead? I think this should work:
```suggestion
      url: '/api/v2/help_center/categories/{category_id}/articles',
      queryParams: {
        include: 'translations',
      },
```
(and also `sort_by: 'updated_at'`) if we do need it)

Reply:
ok - so let's document why this is used, so that if in the future we want to see if we can remove it we will know what to check (and that this is not sorted this way because of something that relies on it in our code)?

Reply:
query params did not work. we said we will try to fix it later.